dotCMS/src/main/java/org/apache/velocity/tools/view/context/ChainedContext.java

```diff
@@ -87,31 +86,34 @@ public ChainedContext(Context ctx,
                           HttpServletResponse response,
                           ServletContext application)
     {
-        this(ctx, null, request, response, application);
+        this(ctx, null, request, response);
     }
 
 
     public ChainedContext(VelocityEngine velocity,
                           HttpServletRequest request,
-                          HttpServletResponse response,
-                          ServletContext application)
+                          HttpServletResponse response)
     {
-        this(null, velocity, request, response, application);
+        this(null, velocity, request, response );
+    }
+    public ChainedContext(VelocityEngine velocity,
+                    HttpServletRequest request,
+                    HttpServletResponse response,ServletContext application)

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'response' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5378571)